scripts/docker-bootnode/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN go install -tags=prod \
     github.com/athanorlabs/atomic-swap/cmd/swapcli@"${VERSION}"
 RUN /go/bin/bootnode --version
 
-FROM debian:bullseye-slim
+FROM debian:bookworm-slim
```

Comment:
The Go docker image that we build the executables in is now based off of bookworm (debian 12) and glibc versions are only backwards compatible, not forwards. We could built the bootnode executable statically, but in this case I was updating a bootnode when I discovered the issue, so maybe keeping the `swapd`/`bootnode` images close to each other is more of a win.